components/picker/picker.component.ts

```diff
@@ -20,7 +20,7 @@ import { takeUntil } from 'rxjs/operators';
 import { Subject } from 'rxjs';
 
 @Component({
-  selector: 'Picker, nzm-picker',
+  selector: 'Picker',
```

Comment:
Why delete nzm-picker ？

Reply:
it is redundancy code， we do not use selector to show picker any more